test/api.js

```diff
@@ -526,6 +527,30 @@ describe('Duo API', function () {
       yield duo.run();
     });
 
+    it('should update dependencies when the manifest changes', function *() {
+      var duo = build('manifest-modify');
+
+      var a = path('manifest-modify', 'component-a.json');
+      var b = path('manifest-modify', 'component-b.json');
+      var manifest = path('manifest-modify', 'component.json');
+
+      // write manifest a
+      yield fs.writeFile(manifest, yield fs.readFile(a, 'utf8'));
+      var adeps = yield duo.install();
+
+      yield wait(1000); // need to wait so mtime changes
```

Comment:
Haha, yeah I needed to be positive this test wouldn't fail because of a race-condition.

Since unix systems only register the `mtime` in seconds, (not milliseconds) the comparison later only succeeds if the mtime is at least 1 full second apart. :(